packages/cdk8s-plus-17/test/config-map.test.ts

```diff
@@ -1,6 +1,15 @@
 import { Testing } from 'cdk8s';
+import { Node } from 'constructs';
 import { ConfigMap } from '../src';
 
+test('defaultChild', () => {
+
+  const chart = Testing.chart();
+
+  expect(Node.of(new ConfigMap(chart, 'ConfigMap')).defaultChild).toBeTruthy();
```

Comment:
would it be better to assert `instanceof KubeConfigMap` or at least `ApiObject`.

BTW, should we rename `ApiObject` to `KubeObject`?